mtp_prisoner_location_admin/apps/prisoner_location_admin/forms.py

```diff
@@ -19,6 +19,10 @@ def __init__(self, *args, **kwargs):
 
     def clean_location_file(self):
         locations = []
+
+        if not self.cleaned_data['location_file'].name.endswith('.csv'):
+            raise forms.ValidationError("Uploaded file must be of CSV type")
```

Comment:
Should this be translatable? Also, the wording's a little clunky; perhaps “Uploaded file must be a CSV, and have a .csv extension”? @leighmoney, you got any suggestions?